doc/changelog.rst

```diff
@@ -8,16 +8,30 @@ To be released at some future point in time
 
 Description
 
+- Tweak the build system to enable building SmartRedis with Nvidia's toolchain
+- Improvements/upgrades to the container used for Github actions
+- Rework code to avoid compiler warnings
 - Added developer documentation on how to run a single test case and eliminated duplicative environment variables
 - Resolve a linting issue with pybind-to-python error propagation
 - Use mutable fields to enable Dataset get methods that store memory to be marked const
 
 Detailed Notes
 
+- Change the order of arguments in our MakeFile to ensure that all dependencies are compiled with GCC (PR448_)
+- Add new user-configurable parameters DEP_CC, DEP_CXX to control which compiler is used to build dependencies (PR448_)
+- Bump the container version to Ubuntu 22.04 to be able to start testing GCC 12 (PR448_)
+- Bump the versions of GCC used to only the currently maintained versions (PR448_)
```

Comment:
Restrict? And this should say CI/CD in it to keep a customer from thinking we're not letting them use gcc8 any more